syntax = "proto3";

package modelpredict;

service ModelPredict {
    rpc Predict(Usage) returns (Choice) {}
}

message Usage {
    double cpuUsage            = 1;
    double memeoryUsage        = 2;
    string otherRules          = 3;
    string podName             = 4;
}

message Choice {
    string nodeName            = 1;
}